jdk/make/closed/autoconf/generated-configure.sh

```diff
@@ -36813,7 +36813,11 @@ $as_echo "$OUTPUT_DIR_IS_LOCAL" >&6; }
 # At the end, call the custom hook. (Dummy macro if no custom sources available)
 
   # Add the J9VM vm lib directory into native LDFLAGS_JDKLIB path
-  LDFLAGS_JDKLIB="${LDFLAGS_JDKLIB} -L${JDK_OUTPUTDIR}/../vm"
+  if test "x$OPENJDK_BUILD_OS_ENV" = xwindows.cygwin; then
+    LDFLAGS_JDKLIB="${LDFLAGS_JDKLIB} -libpath:${JDK_OUTPUTDIR}/../vm/lib"
```

Comment:
Not sure why the directory is different from other platforms.

Reply:
it is on win

Reply:
I realize its on Windows, but what is it about Windows that makes the directory different?

Reply:
Seems on Windows you link to the .lib files which are in the lib subdirectory, while on other platforms you link to the .so files which are in the root vm directory.